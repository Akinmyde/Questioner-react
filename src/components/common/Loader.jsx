import React from 'react';

const Loader = () => (
  <div id="overlay">
    <div id="loader" />
  </div>
);

export default Loader;
