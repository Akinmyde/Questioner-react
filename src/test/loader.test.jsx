import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../components/common/Loader';

describe('HomePage component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div'))
  });
});
