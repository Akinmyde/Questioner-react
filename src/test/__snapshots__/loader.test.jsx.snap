// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should match snapshot 1`] = `
<div
  id="overlay"
>
  <div
    id="loader"
  />
</div>
`;
